lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -194,7 +194,10 @@ public APIGatewayProxyResponseEvent handleRequest(
                 }
             }
 
-            Boolean reproveIdentity = claimsSet.getBooleanClaim(REPROVE_IDENTITY_KEY);
+            Boolean reproveIdentity =
+                    configService.enabled(CoreFeatureFlag.REPROVE_IDENTITY_AUDIT_ENABLED)
```

Comment:
I think this is only parsing the claim for the audit event? It's also saved into the client session (line 174 `clientOAuthSessionService.generateClientSessionDetails(..`) so we need to feature flag the parsing inside there too